src/monitor/formation_metadata.c

```diff
@@ -656,6 +656,11 @@ FormationNumSyncStandbyIsValid(AutoFailoverFormation *formation,
 	List *standbyNodesGroupList = NIL;
 	int count = 0;
 
+	if (formation == NULL)
+	{
+		return false;
```

Comment:
I think we should error out here as well

Reply:
I think we can also add error code for these two errors, probably `errcode(ERRCODE_INVALID_PARAMETER_VALUE)`